src/gt4py/next/iterator/transforms/fuse_as_fieldop.py

```diff
@@ -83,7 +87,12 @@ def _inline_as_fieldop_arg(
 
     for inner_param, inner_arg in zip(stencil.params, inner_args, strict=True):
         if isinstance(inner_arg, itir.SymRef):
-            stencil_params.append(inner_param)
+            if inner_arg.id in extracted_args:
+                assert extracted_args[inner_arg.id] == inner_arg
+                alias = stencil_params[list(extracted_args.keys()).index(inner_arg.id)]
+                stencil_body = im.let(inner_param, im.ref(alias.id))(stencil_body)
+            else:
+                stencil_params.append(inner_param)
```

Comment:
This is a fix for the case covered in [test_trivial_same_arg_twice](https://github.com/GridTools/gt4py/pull/1764/files#diff-b27471e57df42d67a9d9c781f66431cdcd867d6d0cccb5de6b372466c8136147R58)